po/ja.po

```diff
@@ -9,32 +9,35 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2019-12-18 19:13+0000\n"
-"PO-Revision-Date: 2019-09-20 11:25+0000\n"
-"Last-Translator: Copied by Zanata <copied-by-zanata@zanata.org>\n"
-"Language-Team: Japanese\n"
+"POT-Creation-Date: 2020-01-15 11:50+0000\n"
+"PO-Revision-Date: 2020-01-14 19:25+0000\n"
+"Last-Translator: Ludek Janda <ljanda@redhat.com>\n"
+"Language-Team: Japanese <https://translate.stg.fedoraproject.org/projects/"
+"cockpit/master/ja/>\n"
 "Language: ja\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=1; plural=0\n"
-"X-Generator: Zanata 4.6.2\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
```

Comment:
So this semicolon is the real issue here. I opened issue about it: https://github.com/mikeedwards/po2json/issues/87
I am not sure though, if it gets fixed/gets fixed in some reasonable amount of time. Maybe we can add some hack into bots, that would strip this semicolon? (it works always without it and we even add it back there in our `po/po2json` script).

Reply:
Or could it be, that this was known issue for long time and Zanata was not updating this and nor is Weblate, but when @ludekjanda pushed `.po` files they contained this semicolon?

Reply:
Ok, this is on Weblate - https://github.com/cockpit-project/cockpit-weblate/commit/fa85b89a11cd9b6d6eeded161252f54e30f02094